awsutil_v2/awsutil_v2.go

```diff
@@ -0,0 +1,187 @@
+package awsutil_v2
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/mitchellh/mapstructure"
+)
+
+// Some AWS destinations are using SecretAccessKey instead of accessKey
+type SessionConfig struct {
+	Region              string         `mapstructure:"region"`
+	AccessKeyID         string         `mapstructure:"accessKeyID"`
+	AccessKey           string         `mapstructure:"accessKey"`
+	SecretAccessKey     string         `mapstructure:"secretAccessKey"`
```

Comment:
What's the diff btw `AccessKey` & `SecretAccessKey` ? We have this because rudderstack is using these keys for different destinations?